device_doctor/lib/src/utils.dart

```diff
@@ -0,0 +1,461 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
This file seems like a catch all. Can it be split into it's various pieces (process helpers, health interfaces, config)

Reply:
Re-organized the code. Please take another look.

Reply:
A lot of the functionality of this file is not needed in the context of this tool can we do a cleanup?